packages/layout/src/components/cards/actuary/i18n.ts

```diff
@@ -16,6 +16,7 @@ export type ActuaryI18nProps = {
 	};
 	name: {
 		title: string;
+		sectionTitle: string;
```

Comment:
should the `sectionTitle` prop be optional here too? (and for all other cards)
just to prevent breaking the existing app where the existing i18n data does not include `sectionTitle`.

Reply:
Thanks, good spot

Reply:
Made all `sectionTitle` types in `i18n` optional